### PR TITLE
Scipy toolkit

### PR DESCRIPTION
**Pull request by pkohvaei**
Ensemble estimator tool added.

### CODE REVIEW

**Comment by bgruening** [comment]
Incredible PR! Nice work!
Just one thing I have seen, why do you have two statistic-tools? Is one of them deprecated?

**Comment by bgruening** [comment]
"Ran 67 tests in 1326.089s" :)
Can you please add a `.shed.yml` to `tools/statistics` this will also trigger tests for these tools.

File: tools/statistics/macros.xml [hunk 1]
```diff
@@ -0,0 +1,262 @@
+<macros>
+  <token name="@VERSION@">0.2</token>
+
+  <xml name="python_requirements">
+    <requirements>
+        <requirement type="binary">@EXECUTABLE@</requirement>
```
Comment (bgruening) [inline tools/statistics/macros.xml]: I don't think this is needed anymore.

**Comment by pkohvaei** [comment]
Those are not complete yet. But I will add the yml file.

File: tools/statistics/macros.xml (hunk 1 above)
Comment (pkohvaei) [inline tools/statistics/macros.xml]: It is still using the python script. Next step that would be removed.

**Pull request merged**